source/includes/steps-define-custom-user-data-import-export.yaml

```
@@ -1,106 +1,50 @@
+     
      {
-       ...,
-       "custom_user_data_config": {
-         "enabled": <boolean>,
-         "mongo_service_id": "<Linked MongoDB Cluster Service ID>",
-         "database_name": "<Database Name>",
-         "collection_name": "<Collection Name>",
-         "user_id_field": "<User ID Field Name>"
-       }
+       "enabled": <Boolean>,
+       "mongo_service_name": "<MongoDB Data Source Name>",
+       "database_name": "<Database Name>",
+       "collection_name": "<Collection Name>",
+       "user_id_field": "<Field Name>"
      }
-
-  .. list-table::
-     :header-rows: 1
-     :widths: 20 80
-     
-     * - Field
-       - Description
-     
-     * - ``enabled``
-       - A boolean that, if ``true``, indicates that {+service-short+} should
-         associate data in the specified collection with application
-         users.
-     
-     * - ``mongo_service_id``
-       - The service ID of the :doc:`linked MongoDB cluster </mongodb>`
-         that contains the custom user data collection.
-     
-     * - ``database_name``
-       - The name of the MongoDB database that contains the custom user
-         data collection.
-     
-     * - ``collection_name``
-       - The name of the MongoDB collection that contains custom user
-         data.
-     
-     * - ``user_id_field``
-       - The name of the field that maps each document to a specific application
-         user. The field must have the same name in each document and contain
-         the user's ID as a string.
-         
-         .. note::
-            
-            If two documents contain the same user ID, {+service-short+} only
-            maps the first matching document to the user.
 ---
-title: Deploy the Updated Application
-ref: deploy-the-updated-application
+title: Deploy the Custom User Data Configuration
+ref: deploy-the-custom-user-data-configuration
 content: |
-  Once you have configured the custom user data collection, you can
-  make custom user data available to client applications by deploying
-  your application.
-
-  To deploy a draft application with {+cli-bin+}:
-
-  .. code-block:: shell
-
-     realm-cli import
-
-  To deploy a draft application with automatic :doc:`GitHub deployment
-  </deploy/deploy-automatically-with-github>`:
-
-  .. code-block:: shell
-
-     git add config.json
-     git commit -m "Configure and Enable Custom User Data"
-     git push origin <branch name>
-
-  Once the application successfully deploys, {+service+} begins to associate
-  custom data with users. When a user logs in, {+service-short+} automatically
-  queries the custom user data collection for a document where the
-  specified :guilabel:`User ID Field` contains the user's ID. If a
-  document matches, {+service-short+} exposes the data in the document in the
-  ``custom_data`` field of that user's :ref:`user object
-  <user-objects>`.
+  Once you've configured custom user in ``/auth/custom_user_data.json``, you can
```

```suggestion
  Once you've configured a custom user in ``/auth/custom_user_data.json``, you can
```
Should it be 'a custom user' or 'the custom user'

---

Tweaked